daemon/algod/api/algod.oas3.yml

```diff
@@ -2622,6 +2630,10 @@
                       "properties": {},
                       "type": "object",
                       "x-algorand-format": "BlockCertificate"
+                    },
+                    "hash": {
+                      "description": "Hash of current block.",
```

Comment:
```suggestion
                      "description": "Hash of current block. This is not included when the format is set to message pack.",
```

Reply:
Most of the SDKs use the msgpack format, so they wouldn't receive the new field.

Reply:
Only the go SDK has a real block object, I don't know if it has the helper function.

Reply:
I believe if you regenerate the java SDKs with this change you'd end up getting a new hash field but it's never populated. Go has a special object which isn't generated, so it would just ignore the new field. Python & JS don't have typed objects so they would work as expected (if they use json format, I'm not sure what they use)